code/game/objects/effects/portals.dm

```diff
@@ -1,35 +1,102 @@
 /obj/effect/portal
 	name = "portal"
-	desc = "Looks unstable. Best to test it with the clown."
+	desc = "There's a hole in the sky... through which I can fly."

```

Comment:
@clusterfack The first one made sense since you couldn't see through it, but now that you can actually see the destination turf through it and that the chance of failure doesn't happen when you cross it anymore it doesn't do justice to call it "unstable".